reference/functions/mergedata.markdown

```diff
@@ -2,22 +2,28 @@
 layout: default
 title: mergedata
 published: true
-tags: [reference, data functions, functions, json, merge, mergedata, container]
+tags: [reference, data functions, functions, json, merge, mergedata, container, wrap, extract, array, map]
 ---
 
 [%CFEngine_function_prototype(one, two, etc)%]
 
-**Description:** Returns the merger of any named data containers or lists.
+**Description:** Returns the merger of any named data containers or lists.  Can also wrap and unwrap data containers.
 
 The returned data container will have the keys from each of the named
-data containers.
+data containers, arrays, or lists.
 
-If all data containers are JSON arrays, they are merged into a single
-array, as you'd expect from merging two arrays.
+If all the data containers are JSON arrays, they are merged into a
+single array, as you'd expect from merging two arrays.
 
 If any of the data containers are JSON objects, all the containers are
 treated as JSON objects (for arrays, the key is the element's offset).
 
+If a key inside a data container is specified (`container[key]`), the
+value under that key is extracted and merged. The key can be a string
+for JSON objects or a number for JSON arrays. Note that a single thing
+can be named, in which case you're simply extracting the contents of
+the data container into a new data container!
```

Comment:
s/thing/data variable/ (or perhaps container)

And I would replace the ending '!' with simply '.'.